nservicebus/azure/data-distribution.md

```diff
@@ -12,6 +12,8 @@ redirects:
 - nservicebus/scalability-and-ha/individualizing-queues-when-scaling-out
 ---
 
+DANGER: Asynchronous messaging is not an optimal solution for data distribution scenarios. It is usually better to use a dedicated data distribution technology for that purpose, such as a distributed cache or distributed configuration service.
```

Comment:
@yvesgoeleven and @SeanFeldman - Can you review this disclaimer to make sure that it is still accurate when it comes to Azure. Or can you see if we can make this disclaimer better by adding some more, especially when it comes to caching as a service and azure.